checker/src/main/java/org/checkerframework/checker/resourceleak/MustCallConsistencyAnalyzer.java

```diff
@@ -537,7 +557,7 @@ private boolean isValidCreatesMustCallForExpression(
       Element elt = ((LocalVariable) expression).getElement();
       if (!checker.hasOption(MustCallChecker.NO_LIGHTWEIGHT_OWNERSHIP)
           && typeFactory.getDeclAnnotation(elt, Owning.class) != null) {
-        // The expression is an Owning param.  This satisfies case 1.
+        // The expression is an Owning formal parameter or local variable.  This satisfies case 1.
```

Comment:
This comment is now incorrect: `@Owning` annotations cannot be written on local variable declarations (which are always treated as owning), so this case can only be reached for owning formal parameters.